lib/src/components/NxDateInput/stateful/NxStatefulDateInput.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef, useState } from 'react';
+
+import { initialState, userInput } from '../../NxTextInput/stateHelpers';
+import NxDateInput from '../NxDateInput';
+
+import { Props, propTypes } from './types';
+export { Props, propTypes } from './types';
+
+const NxStatefulDateInput = forwardRef<HTMLDivElement, Props>(
```

Comment:
This component has no documentation in the gallery

Reply:
> This component has no documentation in the gallery

Added:
76e109d